docs/content/drafts/PageLayout.mdx

```diff
@@ -104,7 +175,7 @@ description: TODO
 | 'large'
 | 'xlarge'`}
     defaultValue="'full'"
-    description="Define the maximum width of the content region."
+    description="The maximum width of the content region."
   />
   <PropsTableSxRow />
```

Comment:
It seems like `PageLayout.*` do not accept `sx` prop yet. Is it an intentional choice to leave it out?

Reply:
Nope. Just haven't gotten to it yet

Reply:
Added ✅